packages/db/src/definitions/networkGenealogies.ts

```diff
@@ -0,0 +1,124 @@
+import gql from "graphql-tag";
+
+import {Definition} from "./types";
+
+export const networkGenealogies: Definition<"networkGenealogies"> = {
+  createIndexes: [],
+  idFields: ["ancestor", "descendant"],
+  typeDefs: gql`
+    type CandidateSearchResult {
+      network: Network!
+      alreadyTried: [ID]! #will include all networks returned
+    }
+
+    type NetworkGenealogy implements Resource {
+      id: ID!
+      ancestor: Network
+      descendant: Network
+    }
+
+    input NetworkGenealogyInput {
+      ancestor: ResourceReferenceInput!
+      descendant: ResourceReferenceInput!
+    }
+
+    extend type Network {
+      possibleAncestors(
+        alreadyTried: [ID]!
+        limit: Int # will default to 5
+      ): [CandidateSearchResult]!
+      possibleDescendants(
+        alreadyTried: [ID]!
+        limit: Int # will default to 5
+      ): [CandidateSearchResult]!
+    }
```

Comment:
In the same reasoning as above, this could just be part of `type Network {}` in the first place (in `definitions/networks`)